### ISSUE TITLE
dotcms not starts using the empty starter when the default language is changed

### ISSUE THREAD

**Issue by erickgonzalez** [dotCMS/core#18766]
If you change the default language in our **dotmarketing-config.properties** and try to use the empty starter it throws this error: 

```
Caused by: java.lang.NullPointerException
        at com.dotmarketing.portlets.languagesmanager.business.LanguageFactoryImpl.createDefaultLanguage(LanguageFactoryImpl.java:155)
        at com.dotmarketing.portlets.languagesmanager.business.LanguageFactoryImpl.getDefaultLanguage(LanguageFactoryImpl.java:275)
        at com.dotmarketing.portlets.languagesmanager.business.LanguageAPIImpl.getDefaultLanguage_aroundBody20(LanguageAPIImpl.java:182)
```

Steps to Reproduce:

1. Change the default language properties in dotmarketing-config.properties.
```
DEFAULT_LANGUAGE_COUNTRY = Spain
DEFAULT_LANGUAGE_CODE = es
DEFAULT_LANGUAGE_STR = Spanish
DEFAULT_LANGUAGE_COUNTRY_CODE = ES
```
2. Use the empty starter to start up dotcms.

**Comment by ivorpad** [comment]
dotCMS deployed & started successfully with the settings mentioned above.

**Comment by bryanboza** [comment]
Fixed, tested on release-21.06 // Postgres // FF... Tested changing the default to Spanish
